frontend/www/js/omegaup/badge/list.ts

```diff
@@ -0,0 +1,26 @@
+import { OmegaUp } from '../omegaup';
+import { types } from '../api_types';
+import Vue from 'vue';
+
+import badge_List from '../components/badge/List.vue';
+
+OmegaUp.on('ready', function() {
+  const payload = types.payloadParsers.BadgeListPayload();
+  let badgeList = new Vue({
```

Comment:
nit:

```suggestion
  const badgeList = new Vue({
```

si no se re-define alguna variable, es mejor usar `const`.